nanome/setup_config.py

```diff
@@ -1,12 +1,17 @@
 import argparse
 import sys
+import logging
```

Comment:
Why use `logging` at all in this file? Shouldn't `print` do the job?

Reply:
Agreed, I went back and forth on that, but print() will work just fine